synapse/handlers/set_password.py

```diff
@@ -32,30 +34,37 @@ def __init__(self, hs):
         self._device_handler = hs.get_device_handler()
 
     @defer.inlineCallbacks
-    def set_password(self, user_id, newpassword, requester=None):
+    def set_password(
+        self,
+        user_id: str,
+        newpassword: str,
```

Comment:
Small nit: could you fix `newpassword` -> `new_password` while you're here?

Reply:
Done in 25eb472fd73853ace2b8dba86bb994983b39b13f!